models/git/branch.go

```diff
@@ -378,3 +380,22 @@ func RenameBranch(ctx context.Context, repo *repo_model.Repository, from, to str
 
 	return committer.Commit()
 }
+
+// FindRecentlyPushedNewBranches return at most 2 new branches pushed by the user in 3 days which has no opened PRs created
```

Comment:
Wasn't it three in the other PR?
Either way, I'm probably fine with both

Reply:
I changed from 3 to 2 after I created a 3-popup screenshot. I think 2 is more suitable.